File: Examples/CSharp/Articles/CopyRowsColumns/CopyingSingleColumn.cs
using System;
using System.IO;

using Aspose.Cells;

namespace Aspose.Cells.Examples.CSharp.Articles.CopyRowsColumns
{
    public class CopyingSingleColumn
    {
        public static void Run()
        {
            //Source directory
            string sourceDir = RunExamples.Get_SourceDirectory();

            //Output directory
            string outputDir = RunExamples.Get_OutputDirectory();

            // Instantiate a new Workbook
            // Open an existing excel file
            Workbook workbook = new Workbook(sourceDir + "sampleCopyingSingleColumn.xlsx");

            // Get the first worksheet
            Worksheet worksheet = workbook.Worksheets[0];
            // Get the Cells collection
            Cells cells = worksheet.Cells;

            //Copy the first column to next 10 columns
            for (int i = 1; i <= 10; i++)
            {
                cells.CopyColumn(cells, 0, i);
            }

            // Save the excel file
            workbook.Save(outputDir + "outputCopyingSingleColumn.xlsx");

            Console.WriteLine("CopyingSingleColumn executed successfully.");
        }
    }
}

File: Examples/VisualBasic/Articles/CopyRowsColumns/CopyingSingleColumn.vb
Imports Microsoft.VisualBasic
Imports System.IO

Imports Aspose.Cells

Namespace Articles.CopyRowsColumns
    Public Class CopyingSingleColumn
        Public Shared Sub Run()
            ' ExStart:1
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType)

            ' Instantiate a new Workbook
            ' Open an existing excel file
            Dim workbook As New Workbook(dataDir & "aspose-sample.xlsx")

            ' Get the first worksheet
            Dim worksheet As Worksheet = workbook.Worksheets(0)
            ' Get the Cells collection
            Dim cells As Global.Aspose.Cells.Cells = worksheet.Cells
            ' Copy the first column to next 10 columns
            For i As Integer = 1 To 10
                cells.CopyColumn(cells, 0, i)
            Next i
            ' Save the excel file
            workbook.Save(dataDir & "output_out_.xlsx")
            ' ExEnd:1
        End Sub
    End Class
End Namespace